docs/reference/cli.mdx

```diff
@@ -0,0 +1,130 @@
+The Nitric CLI allows you to build, develop, and deploy your serverless application.
+
+To verify your installation run `nitric version`. The output should look something like this:
+
+```
+Go Version: go1.17.8
+Go OS/Arch: darwin/amd64
+Git commit: fce97c9
+Build time: 2022-03-23T14:05:59+1100
+Nitric CLI: v1.1.0-develop.14-98-gfce97c91c5bce6eeb864194e592134322b52d8e5
+```
+
+To get a list of the available commands, run the following command:
+
+```
+nitric help
+```
+
+## Common Commands
+
+There are a few common commands that you will use frequently:
+
+- `nitric new` creates a new project
+- `nitric stack new` creates a new stack within a project
+- `nitric run` runs your stack locally
+- `nitric up` deploys your stack to the cloud
+- `nitric down` destroys your stack from the cloud
+
+## Environment Variables
+
+### Pulumi Access Token
+
+To view your deployments on the pulumi dashboard, you will need a pulumi access token. You can get this token by logging into pulumi on the browser and going to your profile settings. Under the 'Access Tokens' tab click 'Create token'.
+
+Add the environment variable to your `~/.zshrc` or `~/.bashrc` as:
+
+```
+PULUMI_ACCESS_TOKEN=access_token
+```
+
+## Start
+
+### Projects
+
+To start a new project with nitric, run the `nitric new` command.
+
+```
+nitric new
+```
+
+This will walk through creating the project, including the name and what template you want to start from.
+
+The command will also create a `nitric.yaml` file. This file contains the configuration for your project. The handlers key is list of globs for your functions. The handlers should point to all functions you want to run or deploy.
+
+### Stacks
+
+Creating a new stack will create a configuration for a particular cloud. You can have multiple stacks for one project.
+
+```
+nitric stack new
+```
+
+This will create a `nitric-stackName.yaml` file that contains the configuration for deploying to the cloud. Some providers will have different config, heres an example of an aws stack
+
+```yaml
+# nitric-my-aws-stack.yaml
+name: my-aws-stack
+provider: aws
+region: us-east-1
+```
+
+## Development
+
+When you are done developing your application and you want to run and test it locally, you can use the `nitric run` command.
+
+```
+nitric run
+```
+
+This will build and run your functions for local testing. The output will provide a local endpoint so that you can test your apis.
+
+If you make changes whilst it is running, the changes will be reflected due to hot reloading.
+
+## Deploy
+
+Once you have tested your application locally and you're ready to deploy, you can do `nitric up`. This command requires you supply the name of a stack with the `-s` or `--stack` argument. This will set the cloud configuration for that particular deployment.
+
+```
+nitric up --stack my-aws-stack
```

Comment:
perhaps encourage users to use the short flags?
```suggestion
nitric up -s my-aws-stack
```

Reply:
Amended